tutorial/Linux_Tutorial.md

```diff
@@ -15,51 +20,66 @@ mkdir -p $CONFBOT_WS/src && cd $CONFBOT_WS && git clone https://github.com/Karst
 ### Adapting the code for your machine
 ```bash
 cd $CONFBOT_WS
-sed -i s@/Users/karsten/workspace/osrf/roscon2018_ws/@$CONFBOT_WS/@g `grep -lr "/karsten/"  --exclude README.md`
+sed -i s@/Users/karsten/workspace/osrf/roscon2018_ws/@$CONFBOT_WS/@g `grep -lr "/karsten/"  --exclude Linux_Tutorial.md`
 ```
 
 ### Installing the dependencies
 ```bash
 sudo rosdep init
 rosdep update
-rosdep install -y --from-paths src --ignore-src --rosdistro crystal
+rosdep install -y --from-paths src --ignore-src
 ```
 
-### Source crystal environment
+### Build the workspace
 ```bash
-source /opt/ros/crystal/setup.bash
+colcon build --merge-install --cmake-args --no-warn-unused-cli -DSECURITY=ON -DBUILD_TESTING=OFF
```

Comment:
i believe this can be extended with `--symlink-install`

Reply:
I believe colcon has bugs when you use both `--merge-install` and `--symlink-install`, (similar to https://github.com/colcon/colcon-ros/issues/58 IIRC). It may work for the highest level overlay but overall I would prefer to stick to what we do on CI to avoid surprises.

Reply:
strange. I am running that configuration as my default on osx and it works just fine. But I am okay leaving it as is.